mlflow/tracking/_tracking_service/utils.py

```diff
@@ -80,10 +86,10 @@ def _use_tracking_uri(uri: str) -> None:
     global _tracking_uri
     old_tracking_uri = _tracking_uri
```

Comment:
```suggestion
    old_tracking_uri = get_tracking_uri()
```

can we use `get_tracking_uri` here?

Reply:
@harupy Changing this line somehow caused many test failures (probably because env var handling and fallback is different), let me keep this untouched:p